.travis.yml

```diff
@@ -1,20 +1,49 @@
 dist: xenial
 language: python
-python:
-- '3.6'
-- '3.7'
+matrix:
+  include:
+    - name: "Python 3.6 on Xenial Linux"
+      python: '3.6'
+
+    - name: "Python 3.7 on Xenial Linux"
+      python: '3.7'
+
+    - name: "Miniconda 3.6 on Xenial Linux"
+      env: ANACONDA="3.6"
+
+    - name: "Miniconda 3.7 on Xenial Linux"
+      env: ANACONDA="3.7"
 
 before_install:
-- sudo apt-get update
-- sudo apt-get build-dep python-scipy
+- sudo apt-get update;
+- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+    MINICONDA_OS="Linux";  
+    sudo apt-get build-dep python-scipy;
+  else
+    MINICONDA_OS="MacOSX";
```

Comment:
can you put a comment here explaining why we're setting `MINICONDA_OS="MacOSX";`. 

It doesn't appear that we're testing `osx` yet. =)

Reply:
The `MINICONDA_OS` variable is used to grab the OS-specific Miniconda installer file. I wanted to avoid hard-coding the file name where it could be avoided. I like it being explicit that Linux is not the only possible OS.

Reply:
As we haven't gotten to `osx` testing yet, can we remove it for now? It might be confusing for other future contributors to see this artifact and get the impression that we are testing `osx` but aren't yet. 

Does that make sense?